### PR TITLE
Change 'accredited body' to 'accredited provider'

### PR DESCRIPTION
**Pull request by simonwhatley**

### CODE REVIEW

**Comment by simonwhatley** [comment]
[Change 'accredited body' to 'accredited provider'](https://trello.com/c/Aa6ulM3F/1074-change-accredited-body-to-accredited-provider)

File: app/controllers/accredited-bodies.js [hunk 1]
```diff
@@ -126,13 +126,13 @@ exports.edit_accredited_body_permissions_post = (req, res) => {
...
 
-    req.flash('success', 'Accredited body permissions updated')
+    req.flash('success', 'Accredited provider permissions updated')
     res.redirect(`/organisations/${req.params.organisationId}/cycles/${req.params.cycleId}/accredited-bodies`)
   }
 }
 
 /// ------------------------------------------------------------------------ ///
-/// NEW ACCREDITED BODY
+/// NEW ACCREDITED PROVIDE
```
Comment (Kateharries) [inline app/controllers/accredited-bodies.js:135]: There's a typo here. Says 'Provide' not 'Provider'

**Review by Kateharries — APPROVED** [review]
I just spotted a typo on line 135 but all good apart from that

**Review by victoriawinlow — APPROVED** [review]
All good, thanks

File: app/controllers/accredited-bodies.js (hunk 1 above)
Comment (simonwhatley) [reply app/controllers/accredited-bodies.js:135]: Eagle eyes. In this instance, it's just a code comment. But I'll update.

**Pull request merged**